### PR TITLE
Things from 10.9.1 branch

### PR DESCRIPTION
**Pull request by phil-davis**
## Description
These are the things from the `release-10.9.1` branch that are not yet already in `master`, without the manual changes to `CHANGELOG.md`

Note: the real code changes for 10.9.1 have already been forward-ported to `master` - so there is not actually much diffs in this PR - mostly administrative changes.

The 10.9.1 merge-back PR #39625 has conflicts, so we could merge this PR instead.

## How Has This Been Tested?
CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -378,8 +378,8 @@ changelog:
 	@echo This file is autogenerated on the master branch.
 	@echo Committing changes to this file will result in merge conflicts.
 	@echo ======================================================================
-	docker run --rm -v $(work_dir):$(work_dir) -w $(work_dir) toolhippie/calens >| CHANGELOG.md
-	docker run --rm -v $(work_dir):$(work_dir) -w $(work_dir) toolhippie/calens -t changelog/CHANGELOG-html.tmpl >| CHANGELOG.html
+	docker run --rm -v $(work_dir):$(work_dir) -w $(work_dir) toolhippie/calens calens >| CHANGELOG.md
+	docker run --rm -v $(work_dir):$(work_dir) -w $(work_dir) toolhippie/calens calens -t changelog/CHANGELOG-html.tmpl >| CHANGELOG.html
```
Comment (phil-davis) [inline Makefile:382]: This was done in https://github.com/owncloud/core/pull/39625/commits/5627304b42fabca245733443a3078f64347a26ea

Does this fix or help something?
Is it wanted in `master`?

**Comment by phil-davis** [comment]
@micbar @jnweiger what is happening aboujt merge-back of 10.9.1 to master?

File: Makefile (hunk 1 above)
Comment (phil-davis) [reply Makefile:382]: @micbar I think this came from a commit of yours?
Can you comment please.

Comment (micbar) [reply Makefile:382]: webhippie changed the entrypoint script of the docker image. Before it was implicitly calling the `calens` executable, now we need to do it explicitly

**Review by JammingBen — APPROVED** [review]

**Comment by phil-davis** [comment]
The generated https://github.com/owncloud/core/blob/master/CHANGELOG.md in master looks good.
```
Changelog for ownCloud Core [10.9.1] (2022-01-12)
=======================================
The following sections list the changes in ownCloud core 10.9.1 relevant to
ownCloud admins and users.

[10.9.1]: https://github.com/owncloud/core/compare/v10.9.0...v10.9.1
...
```

The release version and date and... are correct in master.

**Pull request merged**